core/templates/services/schema-default-value.service.ts

```diff
@@ -21,18 +21,73 @@ import { Injectable } from '@angular/core';
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { LoggerService } from 'services/contextual/logger.service';
 
+interface BoolSchema {
+  type: 'bool';
+}
+
+interface UnicodeSchema {
+  type: 'unicode';
+  choices?: string[];
+}
+
+interface HtmlSchema {
+  type: 'html';
+  choices?: string[];
+}
+
+interface IntSchema {
+  type: 'int';
+  choices?: number[];
+}
+
+interface FloatSchema {
+  type: 'float';
+  choices?: number[];
+}
+
+interface ListSchema {
+  type: 'list';
+  items: Schema | Schema[];
```

Comment:
Why not only `Schema[]`?

Reply:
Because if you see here in the list of valid schemas
https://github.com/oppia/oppia/blob/804751e8a43285e5b88e604330a66f4481856278/schema_utils_test.py#L408-L412

`items` can be either list of schemas or only a single schema.